src/coordinator.js

```diff
@@ -5,7 +5,19 @@ import './environment';
 import logger from './utils/logger';
 import { raceTo } from './utils/lifecycle';
 
-const WORKER_COUNT = os.cpus().length - 1 || 1;
+export function getWorkerCount(cpus) {
+  const realCount = os.cpus().length;
+
+  if (cpus === 0) {
+    return realCount - 1 || 1;
+  }
+
+  if (cpus > realCount) {
+    logger.info(`Specified CPU count of ${cpus} is bigger than OS reported count of ${realCount}`);
+  }
+
+  return cpus - 1;
```

Comment:
Out of curiosity, why do you do the -1 behavior anyway? Is that an attempt to leave one CPU free for the main thread or something?